docs/query-builder.md

```diff
@@ -448,64 +448,161 @@ DB::collection('items')
 
 **Push**
 
-Add items to an array.
+Add one or multiple values to the items array.
 
 ```php
+// Push the value to the matched documents
 DB::collection('users')
-    ->where('name', 'John')
-    ->push('items', 'boots');
+  ->where('name', 'John')
+  // Push a single value to the items array
+  ->push('items', 'boots');
+// Result:
+// items: ['boots']
+
+DB::collection('users')
+  ->where('name', 'John')
+  // Push multiple values to the items array
+  ->push('items', ['hat', 'jeans']);
+// Result:
+// items: ['boots', 'hat', 'jeans']
 
+// Or
+
+// Push the value to an exact document
 $user->push('items', 'boots');
+$user->push('items', ['hat', 'jeans']);
```

Comment:
> Push the value to an exact document

The "exact document" language isn't very clear to me. I assume the difference here is that it's referring to calling `push()` on a model object instead of executing an update through the collection.

Would "Push the values directly to a model object" read better?

Reply:
Sorry, I'm not that good in English to write docs😅